app/assets/stylesheets/comp-content.scss

```diff
@@ -0,0 +1,276 @@
+.content {
+
+  @mixin circle($size) {
+    color: $color_text;
+    font-weight: 400;
+    border: 1px solid #00909E;
+    border-radius: 100%;
+    font-size: 0.775em;
+    text-align: center;
+    margin-right: 0.25em;
+    width: $size;
+    height: $size;
+    line-height: $size;
+    display: inline-block;
+  }
+
+  .gradient-bg {
+    background-image: linear-gradient(-180deg, #ecfafc 0%, white 50%);
+    margin-bottom: 5em;
+
+    .inner {
+      max-width: 650px;
+      margin: 0 auto;
+      padding: 1em;
+      padding-top: 5em;
+
+      @media (min-width: 768px) {
+        padding: 0;
+        padding-top: 5em;
+      }
+
+      h1 {
+        font-weight: 600;
+        font-size: 2.25em;
+        margin-bottom: 0.25em;
+      }
+      p {
+        font-weight: 300;
+        color: $color_main;
+        font-size: 1.25em;
+        margin-top: 0;
+        line-height: 1.3;
+      }
+      ul {
+        list-style: none;
+        padding-left: 0;
+      }
+    }
+  }
+
+  article {
+    text-align: center;
+
+    & > * {
+      text-align: left;
+      margin: 1em auto;
+      max-width: 630px;
```

Comment:
I don't think it's a good practice to establish the width of every element. Isn't it simpler to specificy the width of the column? And then if you want the images to stand out, you can force them.